Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,6 +1,15 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "FMDB",
```

Comment:
Could you please check whether we also need to add FMDB to https://github.com/ecosia/ios-browser/blob/main/Client/Assets/About/Licenses.html ? Snowplow is already there but I haven't added FMDB yet